chia/rpc/wallet_rpc_api.py

```diff
@@ -1902,6 +1929,37 @@ async def nft_calculate_royalties(self, request) -> EndpointResult:
             {asset["asset"]: uint64(asset["amount"]) for asset in request.get("fungible_assets", [])},
         )
 
+    async def nft_sign_message(self, request) -> EndpointResult:
+        """
+        Given a NFT ID, sign the message by the NFT's private key.
+        :param request:
+        :return:
+        """
+        if request["nft_id"].startswith("nft"):
```

Comment:
It would be nice to use the `AddressType` utils to avoid hard-coding bech32m prefixes.

```suggestion
        if is_valid_address(request["nft_id"], {AddressType.NFT), self.service.config):
```